dogowner/migrations/0002_test_data.py

```diff
@@ -9,18 +10,52 @@ class Migration(migrations.Migration):
     def generate_data(apps, schema_editor):
         from dogowner.models import DogOwner
 
-        test_data = [
-            ('user1@address.com', 'testUser1', 'password123', 'dog name', 'first name', 'last name',
-             1234567890, 'dog race', 'https://www.google.com/user1', 4, 2, 'M'),
-            ('user2@address.com', 'testUser2', 'password123', 'dog name', 'first name', 'last name',
-             1234567890, 'dog race', 'https://www.google.com/user2', 3, 1, 'M'),
-            ('user3@address.com', 'testUser3', 'password123', 'dog name', 'first name', 'last name',
-             1234567890, 'dog race', 'https://www.google.com/user3', 2, 5, 'M'),
+        static_dog_owners = [
```

Comment:
MIgration files should not be changed manually

Reply:
Got it .